### DISCUSSION TITLE
Test

### DISCUSSION THREAD

**Discussion by johanmalm** [labwc/labwc#832] [Team Posts]
Just trying out the discussion feature with the Compositor Team

**Comment by Consolatis** [comment]
such magic

On a more serious note: just now discovered this feature.

Testing some random issue:
https://github.com/labwc/labwc/issues/280
which is #280
which is in repo labwc#280 or #labwc/280 or #labwc280.
which is also labwc/labwc#280

I am not very convinced of the usefulness of this discussion board in comparison to just PRs/Issues.

**Comment by johanmalm** [comment]
Maybe useful for more general discussions?

**Comment by Consolatis** [comment]
I'd prefer those general discussions to be public as well instead of hidden like this seems to be. People interested in specific decisions should be able to express their own opinion and see what caused them.

To be clear I am not suggesting to disable this org discussion forum I just don't see it being useful all that much.

What we could do here however is to recommend a small setup for new org contributors like linking some IRC clients / tutorials / what-is-a-bouncer and recommend to register the nick at Libera.

Or further define some roles here like to list you as being responsible for releases and release decisions or stuff like that.
